kinode/packages/app_store/install/src/lib.rs

```diff
@@ -34,13 +34,13 @@ fn init(our: Address) {
     let Ok(Ok(Message::Response { body, .. })) =
         Request::to((our.node(), ("main", "app_store", "sys")))
             .body(
-                serde_json::to_vec(&LocalRequest::Install(
-                    crate::kinode::process::main::PackageId {
-                        package_name: package_id.package_name.clone(),
-                        publisher_node: package_id.publisher_node.clone(),
-                    },
-                ))
-                .unwrap(),
+                b"temp", // serde_json::to_vec(&LocalRequest::Install(
+                        //     crate::kinode::process::main::PackageId {
+                        //         package_name: package_id.package_name.clone(),
+                        //         publisher_node: package_id.publisher_node.clone(),
+                        //     },
+                        // ))
+                        // .unwrap(),
```

Comment:
how do you want to implement this? add version hash to the command like for download?

Reply:
yep, that's the plan